forage-android/src/main/java/com/joinforage/forage/android/ecom/services/ForageSDK.kt

```diff
@@ -527,18 +529,18 @@ class ForageSDK {
  * to create multiple payments, set to true by default.
  */
 data class TokenizeEBTCardParams(
-    val foragePanEditText: ForagePanElement,
+    val foragePanEditText: ForagePANEditText,
```

Comment:
I may have missed this in previous PR reviews/commit history, but is this supposed to be `val foragePanEditText: ForagePANEditText,` or `val foragePanEditText: ForagePanElement`

@devinmorgan ?

Reply:
What you have, `ForagePANEditText`, works great! Either is fine imo. The tradeoffs around this are relatively minor